db/db/migrations/20230307094014_orders_item.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE orders_item (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  quantity INT NOT NULL,
+  products_option_id INT NOT NULL,
+  orders_id INT NOT NULL,
+  orders_status_id INT NOT NULL,
+  CONSTRAINT orders_item_products_option_products_option_id_fkey FOREIGN KEY (products_option_id) REFERENCES products_option(id)
+  CONSTRAINT orders_item_orders_orders_id_fkey FOREIGN KEY (orders_id) REFERENCES orders(id)
+  CONSTRAINT orders_item_orders_status_orders_status_id_fkey FOREIGN KEY (orders.status_id) REFERENCES orders.status(id)
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
```suggestion
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  CONSTRAINT orders_item_products_option_products_option_id_fkey FOREIGN KEY (products_option_id) REFERENCES products_option(id),
  CONSTRAINT orders_item_orders_orders_id_fkey FOREIGN KEY (orders_id) REFERENCES orders(id),
  CONSTRAINT orders_item_orders_status_orders_status_id_fkey FOREIGN KEY (orders.status_id) REFERENCES orders.status(id)
```
컬럼에 대한 정의를 먼저 작성 한 후, 테이블의 제약 설정을 작성하는 순서로 맞춰주세요~ 다른 테이블들도 모두 마찬가지입니다.